README.md

```diff
@@ -117,6 +118,40 @@ If you installed globally, run
 prettier path/to/file.php --write
 ```
 
+### In the Browser
+This package exposes a `standalone.js` that can be used alongside Prettier's own `standalone.js` to make the PHP plugin work in browsers without a compile step.
+
+First, grab both standalone scripts from an npm CDN like [unpkg](https://unpkg.com/):
+
+```html
+<script src="https://unpkg.com/prettier/standalone.js"></script>
+<script src="https://unpkg.com/@prettier/plugin-php/standalone.js"></script>
+```
+
+Then use Prettier with PHP, just like this:
+
+```js
+prettier.format(YOUR_CODE, {
+  plugins: [prettierPlugins.php],
```

Comment:
Wouldn't this be better then though?
```suggestion
  plugins: prettierPlugins,
```

Reply:
I'm not sure that this would work. `[prettierPlugins.php]` is an array while `prettierPlugins` is an object. Have you tried if it performs as expected?

Reply:
I've just tested and it works like a charm. 👍